pkg/logs/schedulers/ad/scheduler.go

```diff
@@ -151,6 +152,13 @@ func (s *Scheduler) toSources(config integration.Config) ([]*sourcesPkg.LogSourc
 	case names.Container, names.Kubernetes, names.KubeContainer:
 		// config attached to a container label or a pod annotation
 		configs, err = logsConfig.ParseJSON(config.LogsConfig)
+	case names.RemoteConfig:
+		if pkgconfig.Datadog.GetBool("remote_configuration.agent_integrations.allow_log_config_scheduling") {
+			// config supplied by remote config
+			configs, err = logsConfig.ParseJSON(config.LogsConfig)
+		} else {
+			log.Warnf("parsing logs config from %v is disabled. You can enable it by setting remote_configuration.agent_integrations.allow_log_config_scheduling to true", names.RemoteConfig)
```

Comment:
Would it be better to make this an error by setting `err = ...` rather than just a warning?

Reply:
Oh sorry, I just saw the conversation above, disregard this comment!

Reply:
This was changed to a warning (from an error)[ by request ](https://github.com/DataDog/datadog-agent/pull/22026#discussion_r1450728780)from the agent-metrics-logs team